ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -1096,6 +1100,80 @@ def test_tensorflow_strided_slice(
             shrink_axis_mask=masks[4],
         )
     except Exception as e:
-        if hasattr(e, 'message'):
+        if hasattr(e, "message"):
             if "only stride 1 allowed on non-range indexing" in e.message:
                 assume(False)
+
+
+@st.composite
+def _linspace_helper(draw):
+    shape = draw(
+        helpers.get_shape(
+            allow_none=False,
+            min_num_dims=0,
+            max_num_dims=5,
+            min_dim_size=1,
+            max_dim_size=10,
+        ),
+    )
+
+    dtype = draw(st.sampled_from(["float32", "float64"]))
+
+    # Param: start
+    start = draw(
+        helpers.array_values(
+            dtype=dtype,
+            shape=shape,
+            min_value=-5.0,
+            max_value=5.0,
+        ),
+    )
+
+    # Param:stop
+    stop = draw(
+        helpers.array_values(
+            dtype=dtype,
+            shape=shape,
+            min_value=-4.0,
+            max_value=10.0,
+        ),
+    )
+
+    return [dtype] * 2, start, stop
+
+
+# linspace
+@handle_frontend_test(
+    fn_tree="tensorflow.linspace",
+    dtype_and_params=_linspace_helper(),
+    num=helpers.ints(min_value=2, max_value=10),
+    axis=helpers.ints(min_value=-1, max_value=0),
+)
+def test_tensorflow_linspace(
+    *,
+    dtype_and_params,
+    num,
+    axis,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    dtype, start, stop = dtype_and_params
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
```

Comment:
please add the `all_aliases=["LinSpace"]` from [here](https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html#alias-functions) as a parameter to this function call in order to test the alias as well! :D

Reply:
Sure, will do that. Apologies on my end for the late reply.

Reply:
Well, after implementing the edit as you said, I'm encountering the following error: 
```E       KeyError: 'LinSpace'```. 

Moreover, the `raw_ops` alternative, i.e `LinSpace` seems to have [different parameters](https://www.tensorflow.org/api_docs/python/tf/raw_ops/LinSpace) than `general_functions` implementation of [linspace](https://www.tensorflow.org/api_docs/python/tf/linspace). So should I implement `LinSpace` in `raw_ops.py` instead of creating an alias or maybe first get the PR merged for the `linspace` `general_functions` implementation which closes #9069? Or this there perhaps something else you'd like to suggest?

Reply:
oh, I see now that the `axis` parameter is not shared by both 😅 

In that case, you can remove all the stuff related to aliases and we can go ahead and merge this PR :)

Reply:
> oh, I see now that the `axis` parameter is not shared by both 😅
yeah, and both start and stop seem to be accepting only 0-D tensors, i.e scalars. 
Anyways, I'll make the requested changes and push asap.

Reply:
@abdrahmandiab I have made the changes and pushed the code. Please have a look at this and let me know if i need to do any further changes. Thanks in advance!

Note: I believe the lint test fails due to the heap of lines i dumped when removing the `LinSpace` alias. I'll shortly make another push to hopefully solve the same. Although I would really appreciate if the pre-commit config had some more hooks in the future :)